airflow-core/tests/unit/cli/commands/test_dag_command.py

```diff
@@ -55,8 +54,14 @@
     clear_db_runs,
     parse_and_sync_to_db,
 )
+from tests_common.test_utils.version_compat import AIRFLOW_V_3_1_PLUS
 from unit.models import TEST_DAGS_FOLDER
 
+if AIRFLOW_V_3_1_PLUS:
+    from airflow.sdk.bases.operator import BaseOperator
+else:
+    from airflow.models.baseoperator import BaseOperator  # type: ignore[no-redef]
```

Comment:
Semantically, we could use v_3_0 here because the move for BaseOperator to sdk was done in that timeframe (not that it will affect the functionality here)

Reply:
I'd rather use try / except actually.